packages/numeric-input/src/NumericInput.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { cx, css } from "emotion";
+
+import { createCustomClassNames } from "@hig/utils";
+import { ControlBehavior } from "@hig/behaviors";
+import Input, { availableVariants } from "@hig/input";
+
+import customStylesheet from "./customStylesheet";
+import SpinnerBehavior from "./behaviors/SpinnerBehavior";
+import SpinnerPresenter from "./presenters/SpinnerPresenter";
+
+export default class NumericInput extends Component {
+  static propTypes = {
+    /**
+     * Adds custom/overriding styles
+     */
+    stylesheet: PropTypes.func,
+    /**
+     * The visual variant of the numeric input
+     */
+    variant: PropTypes.oneOf(availableVariants),
+    /**
+     * Specify if the value of the input is wrong
+     */
+    error: PropTypes.bool
+  };
+
+  static defaultProps = {
+    variant: "line"
+  };
+
+  render() {
+    const {
+      disabled,
+      onBlur,
```

Comment:
seeing as how we only have 3 props exposed, i think all the others should join `className` and come from `otherProps`.